theano/sandbox/cuda/basic_ops.py

```diff
@@ -121,14 +121,7 @@ class GpuFromHost(GpuOp):
 
     check_input = False
 
-    def __eq__(self, other):
-        return type(self) == type(other)
-
-    def __hash__(self):
-        return hash(type(self))
-
-    def __str__(self):
-        return 'GpuFromHost'
+    __props__ = ()
```

Comment:
So you mean, since we access the `props_dict`, that would call the `__props__` of the parent class when we don't have it here, right? I understand